osu.Server.Spectator.Tests/Multiplayer/ModValidationTest.cs

```diff
@@ -0,0 +1,370 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
+using System.Threading.Tasks;
+using Moq;
+using osu.Game.Online.API;
+using osu.Game.Online.Multiplayer;
+using osu.Game.Rulesets.Catch.Mods;
+using osu.Game.Rulesets.Osu.Mods;
+using Xunit;
+
+namespace osu.Server.Spectator.Tests.Multiplayer
+{
+    public class ModValidationTest : MultiplayerTest
```

Comment:
Any reason for not using plurals for class names? As it stands it reads a bit weird. Not using plural would only seem correct if this was testing a class called `ModValidation`, I believe.

Reply:
I haven't done that as it's not usual osu!lazer-side so I wasn't sure, will update them to your way.